misc-utils/Makemodule.am

```diff
@@ -151,6 +151,7 @@ if HAVE_SYSTEMD
 uuidd_LDADD += $(SYSTEMD_LIBS) $(SYSTEMD_DAEMON_LIBS)
 uuidd_CFLAGS += $(SYSTEMD_CFLAGS) $(SYSTEMD_DAEMON_CFLAGS)
 tmpfiles_DATA += misc-utils/uuidd-tmpfiles.conf
+sysusers_DATA += misc-utils/uuidd-sysusers.conf
```

Comment:
The file is deleted below… Is this intentional?

Reply:
Ah, I think it was supposed to be deleted in the first commit, not the third.

Reply:
Yes, fixed.